### DISCUSSION TITLE
https://blog.3place.cn/3place/sszs/SSZS002.html

### DISCUSSION THREAD

**Discussion by giscus** [hbaolong/hbaolong.github.io#105] [General]
# https://blog.3place.cn/3place/sszs/SSZS002.html

https://blog.3place.cn/3place/sszs/SSZS002.html

**Comment by picklecai** [comment]
原来留了个大悬念，最后才发现，三先生就是他爹。这故事真好看。难怪人喜欢看小说。不论写的人写的时候有多艰难，读者看还是不忍释手的。
